.releaserc

```diff
@@ -10,7 +10,7 @@
     [
       "@semantic-release/exec",
       {
-        "successCmd": "npm publish --registry https://registry.npmjs.org"
+        "successCmd": "npm config set @typeform:registry https://registry.npmjs.org/ && npm publish --registry https://registry.npmjs.org"
```

Comment:
That should make the trick. Although we probably don't need the last bit anymore:
```suggestion
        "successCmd": "npm config set @typeform:registry https://registry.npmjs.org/ && npm publish"
```